package/kedro_viz/api/graphql.py

```diff
@@ -49,7 +54,7 @@ class JSONObject(dict):
         description="Generic scalar type representing a JSON object",
     )
 
-
+# TODO: where should format functions go?
```

Comment:
hey so this is what I usually call `serializers` in API development. A serializer takes data from one format (domain model) and serializer it into another format (GraphQL type or FastAPI Response Model) so a file called `graphql_serializers` might be a good idea.

Reply:
Thanks Lim! This is exactly what I was thinking.

My other idea was that it should live in the `Run` type itself. It seems that `strawberry.scalar` has a `serialize` option but `strawberry.type` doesn't, so would need to do this as a method:
```
@strawberry.type
class Run:
    author: Optional[str]
    bookmark: Optional[bool]
    ...

    def format(self):
        ...
```

What do you think? Is this a horrible misunderstanding of the purpose of these objects (maybe there's a good reason that `strawberry.type` doesn't have `serialize`) or a good way to do it?

Reply:
Yea, one down side I can think of is unit testing. Ideally you would want to be able to test the serialization logic which is mostly standalone and stateless without having to mock out the whole API

Reply:
This is what I suspected, thank you for confirming!